src/bin/vip-dev-environment-create.js

```diff
@@ -60,7 +60,19 @@ command()
 
 		debug( 'Args: ', arg, 'Options: ', opt );
 
-		const instanceData = await promptForArguments( opt );
+		let appInfo: any = {};
```

Comment:
This is actually not needed, because `command()` already has an option to fetch an app and env. You just have to pass the GraphQL query to it, and it will return app and env info in `opt`.

Example:

```
command( { 	
  appContext: true,
  appQuery,
  envContext: true, 
} )
```

https://github.com/Automattic/vip/blob/master/src/bin/vip-import-sql.js#L362

Reply:
Yeah, I looked into it, but if I would use it it would require you to provide both app AND environment on every command interaction. Which is not something I wanted. And marking it optional in the command seems like quite a hussle. But that is for sure a way of doing it.